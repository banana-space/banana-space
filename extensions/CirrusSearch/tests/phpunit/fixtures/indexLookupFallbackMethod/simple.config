{
  "query": "hello wolrd",
  "namespaces": [
    0
  ],
  "profile_params": {
    "filters": [
      "filter1",
      "filter2"
    ]
  },
  "offset": 0,
  "profile": {
    "index": "lookup_index_name",
    "params": {
      "bool.filter.0.match.lookup_query_field": "query",
      "bool.filter.1.terms.filtering_field": "params:filters",
      "bool.filter.2.match.wiki": "wiki"
    },
    "query": {
      "bool": {
        "filter": [
          {
            "match": {
              "lookup_query_field": "PLACEHOLDER"
            }
          },
          {
            "terms": {
              "filtering_field": [ "PLACEHOLDER" ]
            }
          },
          {
            "match": {
              "wiki": "PLACEHOLDER"
            }
          }
        ]
      }
    },
    "suggestion_field": "lookup_suggestion_field",
    "metric_fields": []
  }
}
