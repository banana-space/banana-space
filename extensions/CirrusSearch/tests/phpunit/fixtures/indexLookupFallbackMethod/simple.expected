{
    "path": "lookup_index_name\/_search",
    "options": [],
    "query": {
        "query": {
            "bool": {
                "filter": [
                    {
                        "match": {
                            "lookup_query_field": "hello wolrd"
                        }
                    },
                    {
                        "terms": {
                            "filtering_field": [
                                "filter1",
                                "filter2"
                            ]
                        }
                    },
                    {
                        "match": {
                            "wiki": "my_test_wiki"
                        }
                    }
                ]
            }
        },
        "from": 0,
        "size": 1,
        "_source": false,
        "stored_fields": [
            "lookup_suggestion_field"
        ]
    }
}